mlflow/utils/virtualenv.py

```diff
@@ -87,7 +86,9 @@ def _validate_virtualenv_is_available():
 
 
 def _get_pyenv_bin_path():
-    if is_in_databricks_runtime() and os.path.exists(_DATABRICKS_PYENV_BIN_PATH):
+    # We don't check "DATABRICKS_RUNTIME_VERSION" environment variable
```

Comment:
this comment looks confusing now because there is no DATABRICKS_RUNTIME_VERSION in this function